### PR TITLE
Bolt ubi8.4

### PR DESCRIPTION
**Pull request by vikasgupta8**
Build and test package bolt on power architecture ppc64le.

Bolt package is deprecated and was last build in 2017. Hence we are not investigating further for failed tests.

Please refer **notes** file for more details.

### CODE REVIEW

File: b/bolt/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+Apache License
```
Comment (amitsadaphule) [inline b/bolt/LICENSE:1]: Please copy the file instead of the content, so that formatting is not changed.

File: b/bolt/bolt_rhel_ubi8.4.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : bolt
+# Version       : v1.3.1
+# Source repo   : https://github.com/boltdb/bolt
+# Tested on     : RHEL ubi 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : BulkPackageSearch Automation vikas.gupta8@ibm.com
```
Comment (amitsadaphule) [inline b/bolt/bolt_rhel_ubi8.4.sh:8]: Is the Maintainer value supposed to be "BulkPackageSearch Automation vikas.gupta8@ibm.com"?

File: b/bolt/bolt_rhel_ubi8.4.sh [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_PATH=github.com/boltdb/
+PACKAGE_NAME=bolt
+PACKAGE_VERSION=v1.3.1
+PACKAGE_URL=https://github.com/boltdb/bolt
+
+yum -y update && yum install -y wget make
```
Comment (amitsadaphule) [inline b/bolt/bolt_rhel_ubi8.4.sh:23]: Please avoid `yum update` unless absolutely necessary.

File: b/bolt/bolt_rhel_ubi8.4.sh [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+export GOPATH=/home/tester/go
+export GOROOT=/bin/go/
+
+mkdir -p output
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+#export GO111MODULE=on
+
+if ! go get -d -t $PACKAGE_PATH$PACKAGE_NAME; then
```
Comment (amitsadaphule) [inline b/bolt/bolt_rhel_ubi8.4.sh:40]: Should the `PACKAGE_VERSION` be used here?

File: b/bolt/notes [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        	db_test.go:111: page size mismatch
+	SKIP: TestOpen_ErrChecksum (0.00s)
+        	db_test.go:148: page size mismatch
+	SKIP: TestOpen_MetaInitWriteError (0.00s)
+        	db_test.go:339: pending
+
+4. On power architecture ppc64le few tests are taking longer time, more than 1-2 hours. Hence we ran the tests with -timeout 3h option.
+
+As the bolt package was build years ago and deprecated, we are not spending much time to investigate the failure of tests.
+
```
Comment (amitsadaphule) [inline b/bolt/notes:31]: Please consider adding this as part of `README.md`.

File: b/bolt/bolt_rhel_ubi8.4.sh [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+#/home/tester/go/src/github.com/boltdb/bolt
+cd $(ls -d $GOPATH/src/$PACKAGE_PATH$PACKAGE_NAME)
+
+echo `pwd`
+echo "Testing $PACKAGE_PATH$PACKAGE_NAME with $PACKAGE_VERSION"
+
+if ! go test -v -cover -timeout 3h ./...; then
+	echo "------------------$PACKAGE_NAME:install_success_but_test_fails---------------------"
+	echo "$PACKAGE_VERSION $PACKAGE_NAME" > /home/tester/output/test_fails
+	echo "$PACKAGE_NAME  |  $PACKAGE_VERSION | master  | $OS_NAME | GitHub | Fail |  Install_success_but_test_Fails" > /home/tester/output/version_tracker
```
Comment (amitsadaphule) [inline b/bolt/bolt_rhel_ubi8.4.sh:56]: Is `| master  |` really supposed to be a part of this statement?

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

File: b/bolt/notes (hunk 5 above)
Comment (vikasgupta8) [reply b/bolt/notes]: Renamed notes to README.md

File: b/bolt/bolt_rhel_ubi8.4.sh (hunk 6 above)
Comment (vikasgupta8) [reply b/bolt/bolt_rhel_ubi8.4.sh]: Removed master keyword from the log.

File: b/bolt/bolt_rhel_ubi8.4.sh (hunk 3 above)
Comment (vikasgupta8) [reply b/bolt/bolt_rhel_ubi8.4.sh]: Done

File: b/bolt/bolt_rhel_ubi8.4.sh (hunk 2 above)
Comment (vikasgupta8) [reply b/bolt/bolt_rhel_ubi8.4.sh]: Removed BulkPackageSearch Automation.

File: b/bolt/LICENSE (hunk 1 above)
Comment (vikasgupta8) [reply b/bolt/LICENSE]: Replaced the LICENSE file.

File: b/bolt/bolt_rhel_ubi8.4.sh (hunk 4 above)
Comment (vikasgupta8) [reply b/bolt/bolt_rhel_ubi8.4.sh:40]: Go version 1.7.x does not have feature to get specific version. Go version 1.7.x does not have feature of go.mod hence could not get specific version.
the bolt module(v1.3.1) was last build with version 1.7.x. Now bolt module is deprecated and archived in github.

File: b/bolt/bolt_rhel_ubi8.4.sh (hunk 2 above)
Comment (amitsadaphule) [reply b/bolt/bolt_rhel_ubi8.4.sh]: Commonly used format is _NAME <EMAIL>_. Please refer other scripts.

File: b/bolt/bolt_rhel_ubi8.4.sh (hunk 4 above)
Comment (amitsadaphule) [reply b/bolt/bolt_rhel_ubi8.4.sh:40]: OK, do we know which version gets installed using this command then? We'll need to mention that one in the header, if different.

Comment (vikasgupta8) [reply b/bolt/bolt_rhel_ubi8.4.sh:40]: It gets the master branch.

Comment (vikasgupta8) [reply b/bolt/bolt_rhel_ubi8.4.sh:40]: Done. Changes the script to use specified version.

**Review by amitsadaphule — APPROVED** [review]
LGTM

**Pull request merged**